libs/templates/src/lib/home/sections/cta/data.tsx

```diff
@@ -0,0 +1,35 @@
+import { Card, OptimizedImage } from '@deriv-com/components';
+import { Button, qtMerge } from '@deriv/quill-design';
+
+export const CTABlockContent = () => {
+  return (
+    <Card.ContentRight
+      align="start"
+      size="lg"
+      header="Join over 2.5 million online traders"
+      color="gray"
+      content={
+        <>
+          <OptimizedImage
+            className="block lg:hidden"
+            imageName="home/hero/cta_responsive.png"
+            width={712}
+            height={800}
+            alt="cta demo forex mobile"
+          />
+          <OptimizedImage
+            className="hidden lg:block"
+            imageName="home/hero/cta_desktop.png"
+            width={712}
+            height={800}
+            alt="cta demo forex desktop"
+          />
```

Comment:
I think the image optimizer should handle this. On smaller viewports it will already load a smaller size of image so no need to manually handle it from FE side.

Reply:
But design team gave two images, one for desktop and one for mobile.